vroom/clangformat.vroom

```diff
@@ -181,3 +181,10 @@ current buffer.
 
 The -assume-filename arg passed above is also related, used by the clang-format
 tool to detect any custom style rules particular to the current file.
+
+
+Bug #102. When the buffer is empty, clang-format does _not_ return a cursor
+position
+
+  @clear
+  :FormatCode clang-format
```

Comment:
Did you not want to use `@messages (STRICT)` here to catch `Unable to parse clang-format cursor pos`? Or did you find the test already catches the problem as-is if you run it without the fix?

Reply:
this test catches the problem as-is when run without the fix.